index.js

```diff
@@ -91,7 +177,30 @@ function getFormattedDate() {
     min = (min < 10 ? "0" : "") + min;
     sec = (sec < 10 ? "0" : "") + sec;
 
-    var str = '[' + day + "-" + month + "-" + date.getFullYear() + "_" + hour + ":" + min + ":" + sec + ']';
+    const str = `[${day}-${month}-${date.getFullYear()}_${hour}:${min}:${sec}]`;
```

Comment:
new js has Intl, maybe it's relevant instead of moment.js.

[reference](https://dev.to/zenulabidin/moment-js-vs-intl-object-4f8n#:~:text=While%20Moment%20lets%20you%20make,that%20are%20actually%20in%20circulation.)

Reply:
yeah, whatever... just do not format manually. I'm running node v18.12.1 there